mkinstaller.py

```diff
@@ -6,11 +6,22 @@
 
 
 if __name__ == "__main__":
-    repo_type = sys.argv[1]
-    ver = sys.argv[2]
+    if len(sys.argv) != 3:
+        print("Usage: mkinstaller.py <repo_type> <tarantool_version>")
+        sys.exit(1)
+    else:
+        repo_type = sys.argv[1]
+        ver = sys.argv[2]
 
-    with open("installer.tpl.sh", 'r') as file:
-        template = Template(file.read())
+    if ver >= "1" and ver < "3":
+        with open("installer.tpl.sh", 'r') as file:
+            template = Template(file.read())
+    elif ver >= "3":
+        with open("installer-static.tpl.sh", 'r') as file:
```

Comment:
Please check L31. If we use this template, we should open `installer-static.sh` file.